src/DOM/events/delegation.ts

```diff
@@ -19,6 +19,9 @@ export function handleEvent(name, lastEvent, nextEvent, dom) {
 		}
 		if (!lastEvent) {
 			delegatedRoots.count++;
+			if (name === 'onClick') {
```

Comment:
Ideally, we should have a `const` in the scope of this module that detects iOS, rather than apply this all over the place. It will have a performance cost otherwise.